src/bttrx_control.cpp

```diff
@@ -255,6 +279,29 @@ string BTTRX_CONTROL::ParameterTypeToString(ParameterType parameter_type) {
   return return_value;
 }
 
+/**
+ * @brief Set Callsign
+ *
+ * @param callsign
+ * @return ResultType
+ */
+ResultType BTTRX_CONTROL::handleSetCallsign(string callsign) {
+  if (callsign.length() > CALLSIGN_LENGTH) {
+    serial_->dbg_println("Callsign exceed maximum length. Max. length is: " +
+                         CALLSIGN_LENGTH);
```

Comment:
In this case, please `return kError;`

Reply:
Fixed with https://github.com/bt-trx/firmware/pull/40/commits/5927cc77ca56e754ee0d6930ec983344373e2f63